Templates/HttpTrigger-Powershell/run.ps1

```diff
@@ -0,0 +1,8 @@
+$name = Get-Content $Env:req
```

Comment:
I will update batch template as well to make it consistent. However i am not sure how to return 404 for a non Node/C# template. So i am returning hello in case nothing is passed via post body or query string.